src/components/AdminSettings.vue

```diff
@@ -161,6 +161,143 @@
+								t('integration_openproject', 'For automatically managing project folders, this app needs to setup a special group folder, assigned to a group and managed by a user, each called "OpenProject".')
+							}} <br>
+							{{
+								t('integration_openproject', 'The app will never delete files or folders, even if you deactivate this later')
+							}}
+						</p>
+						<GroupFolderError v-if="groupFolderSetupError !== null" :group-folder-set-up-error-message-description="groupFolderSetUpErrorMessageDescription(groupFolderSetupError)" :group-folder-set-up-error="groupFolderSetupError" />
+						<div class="form-actions">
+							<NcButton v-if="groupFolderSetupError === null"
+								type="primary"
+								data-test-id="complete-with-projectfolders-form-btn"
+								@click="setUpProjectGroupFolders">
+								<template #icon>
+									<LoadingIcon v-if="loadingSetupGroupFolder" class="loading-spinner" :size="20" />
+									<CheckBoldIcon v-else :size="20" />
+								</template>
+								{{ textLabelGroupFolderSetupButton }}
+							</NcButton>
+							<NcButton v-else-if="groupFolderSetupError"
+								type="primary"
+								data-test-id="complete-with-projectfolders-form-btn"
+								@click="setUpProjectGroupFolders">
+								<template #icon>
+									<LoadingIcon v-if="loadingSetupGroupFolder" class="loading-spinner" :size="20" />
+									<RestoreIcon v-else :size="20" />
+								</template>
+								{{ t('integration_openproject', 'Retry setup OpenProject user, group and folder') }}
+							</NcButton>
+						</div>
+					</div>
+				</div>
+				<div v-else class="group-folder-status">
+					<div class="group-folder-status-value">
+						<b>Automatic managed folders:</b> {{ state.app_password_set || oPUserAppPassword !== null ? t('integration_openproject', 'Active') : t('integration_openproject', 'Inactive') }}
+					</div>
+					<GroupFolderError
+						v-if="state.app_password_set && !isGroupFolderSetupCorrect"
+						:group-folder-set-up-error-message-description="groupFolderSetUpErrorMessageDescription(state.group_folder_status.errorMessage)"
+						:group-folder-set-up-error="state.group_folder_status.errorMessage" />
+					<div class="form-actions">
+						<NcButton
+							data-test-id="edit-group-folder-setup"
+							@click="setGroupFolderSetUpToEditMode">
+							<template #icon>
+								<PencilIcon :size="20" />
+							</template>
+							{{ t('integration_openproject', 'Edit managed project folders') }}
+						</NcButton>
+					</div>
+				</div>
+			</div>
+		</div>
+		<div v-if="state.app_password_set">
+			<FormHeading index="5"
+				:title="t('integration_openproject', 'Project folders application connection')"
+				:is-complete="isOPSUserAppPasswordFormComplete"
+				:is-disabled="isOPUserAppPasswordInDisableMode" />
+			<div v-if="state.app_password_set">
+				<TextInput v-if="isOPUserAppPasswordFormInEdit"
+					id="openproject-system-password"
+					v-model="oPUserAppPassword"
+					class="py-1"
```

Comment:
Is this class used in styling? Do we need this?
```suggestion
					class="py-1"
```

Reply:
all the components have these. If of no use can be deleted from every component.

Reply:
okay maybe we can check if this is needed and make changes in coming PR's